banners/mobile/C24_WMDE_Mobile_DE_10/components/BannerVar.vue

```diff
@@ -194,6 +197,12 @@ const amountOptionsFive = localFormItemsBuilder.setAmounts( 5, 15, 25, 50, 100 )
 const amountOptionsTen = localFormItemsBuilder.setAmounts( 10, 15, 25, 50, 100 ).getItems().amounts;
 const amountOptionsForForm = ref<FormItem[]>( amountOptionsTen );
 
+const { formAction } = useFormAction(
+	inject<FormActions>( 'formActions' ),
+	10,
+	{ smallAmount: 'xf=0', largeAmount: 'ap=1' }
```

Comment:
should be `xf=1` to activate the "address-opt-out" feature. And we need to check with the PR to ask if we should just show the "normal" form for large amounts (which is `ap=0`( or if we should really show the special email-only form (`ap=1`)